addons/web_editor/static/lib/odoo-editor/test/utils.js

```diff
@@ -298,6 +298,7 @@ export async function testEditor(Editor = OdooEditor, spec, options = {}) {
     } else {
         document.getSelection().removeAllRanges();
     }
+    editor.observerUnactive('beforeUnitTests');
```

Comment:
I suggest to add more details in the commit message about the kind of things which can go wrong when the observers are active. I mean, it worked until now so it's not obvious what can break because of it. Since you encountered at least one particular example, I suggest to document it in the commit message.

Also, could you please change the commit message to: "fix keyboard event in unit tests" ?